RafRate.Core/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RafRate.Core.Interfaces;
+using System;
```

Comment:
Bad tone - leave "using" that are not used in this file 
Fix in all files and in future use this "usings" only when you use something from this library